config/locales/en.yml

```diff
@@ -1279,6 +1279,20 @@ en:
         Hi <<Client.PreferredName>>,
         It looks like you stepped away before finishing our application. Our preparers at <<Client.AssignedOrganization>> look forward to working with you! To complete our form, simply login here: <<Client.LoginLink>>
         Thanks!
+    save_ctc_letter:
+      email:
+        body: |
+          Hello,
+
+          You will receive what’s called a 6419 Letter from the IRS in January 2022. This letter will help you and your tax preparer correctly claim your Child Tax Credit for you and your family. The letter will be have 6419 printed on it and will include information about the Child Tax Credit payments that you received in 2021.
+
+          Please save this letter for when you file your taxes. Whether you file with GetYourRefund or any other service, this will help you collect any missing payments and receive any refund due without delay.
```

Comment:
Do we want "GetYourRefund" to be swapped out with GetCTC if that's what they used before or no?

Reply:
Hmm the copy is the same for that part on both of the copies in the ticket description